arviz/utils.py

```diff
@@ -664,3 +665,32 @@ def credible_interval_warning(credible_interval, hdi_prob):
 
     hdi_prob = credible_interval
     return hdi_prob
+
+
+class HtmlTemplate:
+    """Contains html templates."""
+
+    html_template = """
+            <div>
+              <div class='xr-header'>
+                <div class="xr-obj-type">arviz.InferenceData</div>
+              </div>
+              <ul class="xr-sections">
+              {}
+              </ul>
+            </div>
+            """
+    element_template = """
+            <li class = "xr-section-item">
+                  <input id="idata_{0}" class="xr-section-summary-in" type="checkbox">
+                  <label for="idata_{0}" class = "xr-section-summary">{0}</label>
+                  <div class="xr-section-inline-details"></div>
+                  <div class="xr-section-details">
+                      <ul id="xr-dataset-coord-list" class="xr-var-list">
+                          <div style="padding-left:2rem;">{1}<br></div>
+                      </ul>
+                  </div>
+            </li>
+            """
+    specific_style = ".xr-wrap{width:700px!important;}"
```

Comment:
No change required but just curious what is so specific about this style?

Reply:
The xarray uses the style `min-width:400px; max-width:700px;` for sections. For single xarray dataset max-width is used, while when used in sub-section min-width is used.
So, we are overwriting our own style over xarray's style.

Reply:
Cool!